crops-pipeline/Dockerfile

```diff
@@ -55,7 +55,8 @@ RUN wget https://dl.min.io/client/mc/release/linux-amd64/mc && \
 #Start container as non-root user.
 USER $USER
 
-#Activate the virtual environment in the image.
+#Activate the virtual environment in the image and adjust the path.
 SHELL ["/bin/bash", "-c"]
-RUN source venv/bin/activate
+RUN source venv/bin/activate && \
+    export PATH=$USER_HOME/src/venv/bin:$$PATH
```

Comment:
Every `RUN` command runs in a subshell, so I think export will not work. I think you want

https://stackoverflow.com/a/38742545

the PATH variable will not make it to your actual code

Reply:
Same deal with the `source` . You need to add that to the entrypoint I think

Reply:
If I specify a custom command to the entry point, will the entrypoint still run? If not, then I think I need to run source in the kfp.dsl.ContainerOp and not in the docker file?

Reply:
Maybe putting it in the ContainerOp is the most explicit route, that would make sense to me

Reply:
How does ContainerOp interact with shell? I just noticed that SHELL ["/bin/bash", "-c"] already has the "-c" parameter, maybe that had an effect with the earlier code? Anyway I already have working code so I'll simply move the source and export path to the ContainerOp.